Data/Base/Shaders/Common/Lighting.h

```diff
@@ -74,7 +74,10 @@ float3 SampleSceneColor(float2 screenPosition)
 
 float SampleSceneDepth(float2 screenPosition)
 {
-  float depthFromZBuffer = SceneDepth.SampleLevel(PointClampSampler, float3(screenPosition.xy * ViewportSize.zw, s_ActiveCameraEyeIndex), 0.0f).r;
+  int screenX = clamp(screenPosition.x, 0, ViewportSize.x - 1);
+  int screenY = clamp(screenPosition.y, 0, ViewportSize.x - 1);
```

Comment:
ViewportSize.y, or you just write both lines like:
int2 screenXY = clamp(screenPosition.xy, 0, ViewportSize.xy - 1);
the int4 in load can also be constructed like this
int4(screenXY, s_ActiveCameraEyeIndex, 0)

Reply:
fixed